DistanceRando-Spectrum/DistanceRando.csproj

```
@@ -39,7 +39,7 @@
     <Reference Include="Assembly-CSharp, Version=9.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\Centrifuge.GameSupport.Distance.3.0.7868.41513\lib\net35\Assembly-CSharp.dll</HintPath>
     </Reference>
-    <Reference Include="Centrifuge.Distance, Version=3.0.7868.41513, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="Centrifuge.Distance, Version=3.0.7867.3370, Culture=neutral, processorArchitecture=MSIL">
```

It looks like you may have made a mistake here? The assembly version for the GSL reference is mismatched and as a result, without removing and re-adding the reference, the project won't build for me.
```suggestion
    <Reference Include="Centrifuge.Distance, Version=3.0.7868.41513, Culture=neutral, processorArchitecture=MSIL">
```

---

I don't know why it happened but this shouldn't be a problem